modules/@apostrophecms/doc-type/ui/apos/components/AposDocContextMenu.vue

```diff
@@ -428,13 +428,18 @@ export default {
 
     },
     async customAction(doc, operation) {
+      if (operation.label) {
```

Comment:
How is this related to the label? 🤔

Reply:
Is the idea that label and modal are mutually exclusive? Maybe better to check that `operation.moduleName` is not set before doing this.

Reply:
`operation.moduleName` is set with the wrong module name and is used in some modal, so I guess there is a use case where it's valid. In my use case, I need the current module name.

I've change this `if` condition with a custom property I've created on the backend instead. Will it be safe @boutell ?